mkdocs.yml

```diff
@@ -82,13 +82,14 @@ theme:
     # - navigation.expand
     - navigation.footer
     - navigation.indexes
-    # - navigation.instant
+    - navigation.instant
     # - navigation.instant.prefetch
-    # - navigation.instant.progress
+    - navigation.instant.progress
+    # - navigation.path
     # - navigation.prune
-    - navigation.sections
+    # - navigation.sections
```

Comment:
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections for reference